locales/en-US/webextension.properties

```diff
@@ -15,14 +16,26 @@ notificationLinkCopiedTitle = Link Copied
 # modifier key used in the shortcut (do not translate it): for example, Ctrl-V
 # on Windows systems.
 notificationLinkCopiedDetails = The link to your shot has been copied to the clipboard. Press {meta_key}-V to paste.
+# Error Strings
 requestErrorTitle = Firefox Screenshots is out of order.
-requestErrorDetails = Your shot was not saved. We apologize for the inconvenience. Try again soon.
-connectionErrorTitle = Cannot connect to the Firefox Screenshots server.
-connectionErrorDetails = There may be a problem with the service or with your network connection.
-loginErrorDetails = Your shot was not saved. There was an error authenticating with the server.
-loginConnectionErrorDetails = There may be a problem with the service or your network connection.
-unshootablePageErrorTitle = Page cannot be screenshotted.
-unshootablePageErrorDetails = This is not a normal web page, and Firefox cannot capture screenshots from it.
+requestErrorDetails = Sorry! We couldn’t save your shot. Please try again later.
+connectionErrorTitle = We can’t connect to the Firefox Screenshots server.
+connectionErrorDetails = Please check your network connection. If you are able to connect to the Internet, there may be a temporary problem with the Firefox Screenshots service.
+loginErrorDetails = We couldn’t save your shot because we couldn’t access your account on the Firefox Screenshots server.
+loginConnectionErrorDetails = Please check your network connection. If you are able to connect to the Internet, there may be a temporary problem with the Firefox Screenshots service.
```

Comment:
Yes, it doesn't make a lot of sense to have two separate IDs for the same string. `connectionErrorDetails` sounds good for both.